TestTicTacToe/BoardTests.cs

```diff
@@ -0,0 +1,65 @@
+using TicTacToe;
+using Xunit;
+
+namespace TestTicTacToe
+{
+    public class BoardTests
+    {
+        [Fact]
+        public void Should_Place_Marker_In_The_Correct_Position_If_Empty_Cell()
```

Comment:
I think if you are going to have the condition in this test that the marker is placed in and empty position it would be good to show some sad paths, like you have a board that has the top corner a 1 and you try to place a 1 in that same spot.

Looking at your tests I don't know how you are handling that and one great aspect of having tests is they can act like documentation.

Reply:
This is a simple refactoring issue. 
Initially, this class was validating for duplicate moves, but at a later stage, it was moved out to GameRulesHandler and this test name wasn't updated. Thank you for pointing it out!